src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,31 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
```

Comment:
The class name `FigureSupplier` is good as it is not a verb and describes the purpose of the class well.